index.js

```diff
@@ -38,6 +39,11 @@ function loadAndValidateEnvironment (fastify, opts, done) {
   if (!Array.isArray(opts.data)) {
     data = [data]
   }
+
+  if (opts.dotenv) {
+    require('dotenv').config(Object.assign({}, opts.dotenv))
```

Comment:
now I have a reason to use Object.assign 😛 
It's arguably hacky but I'm relying on Object.assign's behavior of not merging non-enumerables, like `true`